ftp/src/test/scala/akka/stream/alpakka/ftp/CommonFtpStageSpec.scala

```diff
@@ -97,4 +99,47 @@ trait CommonFtpSourceSpec extends BaseSpec {
     }
   }
 
+  "FtpIOSink" should {
```

Comment:
I can easily manage FtpServer permissions with the users.properties file, but still haven't worked out how to do it for SFTP.